tests/PINCacheTests/PINCacheTests.m

```diff
@@ -95,6 +95,28 @@ - (void)testObjectSet
     XCTAssertNotNil(image, @"object was not set");
 }
 
+- (void)testObjectSetWithDuplicateKey
+{
+    NSString *key = @"key";
+    NSString *value1 = @"value1";
+    NSString *value2 = @"value2";
+    __block NSString *value = nil;
```

Comment:
Mind renaming this to something indicating it's the returned value? Maybe cachedValue?